apps/condo/domains/common/components/ExportToExcelActionBar.tsx

```diff
@@ -76,12 +79,12 @@ export const ExportToExcelActionBar: React.FC<IExportToExcelActionBarProps> = (p
                     secondary
                     icon={<DatabaseFilled/>}
                     loading={isXlsLoading}
-                    onClick={handleExportToExcel}
+                    onClick={instrument('ExportExcel', handleExportToExcel)}
```

Comment:
It looks, like we will add this kind of logic everywhere, where we need to track clicks.
Maybe to create our version of `Button` that will wrap button from Ant and will add our custom logic.

From usage point of view it can look like following:

```
import { Button } from `@condo/domains/common/components'
```

```
<Button
    trackEvent="ExportExcel"
    onClick={handleExportToExcel}
/>
```